api/zeebe/generation-strategy.js

```diff
@@ -0,0 +1,28 @@
+const replace = require("replace-in-file");
+
+const outputDir = "docs/apis-tools/zeebe-api-rest/specifications";
+function preGenerateDocs() {
+  // noop
+}
+
+function postGenerateDocs() {
+  removeDuplicateVersionBadge();
+}
+
+module.exports = {
+  outputDir,
+  preGenerateDocs,
+  postGenerateDocs,
+};
+
+function removeDuplicateVersionBadge() {
+  // The generator adds a version badge to the Introduction file, but
+  //   we already have a version badge from the main docs layout.
+  console.log("removing duplicate version badge...");
+  replace.sync({
+    files:
+      "docs/apis-tools/zeebe-api-rest/specifications/zeebe-rest-api.info.mdx",
+    from: /^.*Version: .*$/m,
+    to: "",
+  });
```

Comment:
One thing I'm excited about with this approach is that this step should work cross-platform. The previous implementation of this was using `sed` to remove this line, which works great, but I think it would not have worked on most Windows machines. (I think.)